package.json

```diff
@@ -8,18 +8,26 @@
     "browserify": "^11.2.0",
     "browserify-header": "^0.9.2",
     "chai": "^3.4.1",
+    "chromedriver": "^2.21.2",
     "esdoc-es7-plugin": "0.0.3",
     "express": "^4.13.3",
     "gulp": "^3.9.0",
     "gulp-eslint": "^1.0.0",
+    "gulp-mocha": "^2.2.0",
     "minimist": "^1.2.0",
     "mocha": "^2.3.4",
-    "vinyl-source-stream": "^1.1.0"
+    "require-dir": "^0.3.0",
+    "sinon": "^1.17.3",
+    "vinyl-source-stream": "^1.1.0",
+    "which": "^1.2.4"
   },
   "license": "Apache-2.0",
   "scripts": {
-    "test": "gulp lint --throw-error",
+    "test": "gulp lint --throw-error && gulp && gulp test:automated",
```

Comment:
`gulp lint --throw-error && gulp` can become `gulp --throw-error`, since the default task runs the linter.

And actually, the whole thing can become `gulp default test:automated --throw-error`